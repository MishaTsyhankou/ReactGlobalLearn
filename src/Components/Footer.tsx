import React, { useState } from 'react';

const Footer = () => {
    return (
        <>
            <div className="footer__wrap">
                <div className="logo">
                    <div className="netflix__logo__net">netflix</div>
                    <div className="netflix__logo__roul">roulette</div>
                </div>
            </div>
        </>
    );
};

export default Footer;
