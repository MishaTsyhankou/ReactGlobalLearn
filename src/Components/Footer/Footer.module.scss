.wrapper {
    display: flex;
    justify-content: center;
    background: #424242;
    padding: 20px;
    flex: 0 0 auto;
}
