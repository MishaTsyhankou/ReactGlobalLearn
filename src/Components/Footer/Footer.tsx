import React, { useState } from 'react';
import Logo from '../Netflix/Logo';

import styles from './Footer.module.scss';

const Footer = () => {
    return (
        <>
            <div className={styles.wrapper}>
                <Logo />
            </div>
        </>
    );
};

export default Footer;
