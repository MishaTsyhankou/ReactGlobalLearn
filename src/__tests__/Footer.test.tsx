import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';

import Footer from '../Components/Footer/Footer'


jest.mock('react-router-dom', () => ({
  useNavigate: jest.fn,
}));


describe('<Footer />', () => {
  

  it('should match snapshot', () => {
    const { container } = render(<Footer />);

    expect(container).toMatchSnapshot();
  });

  it('Check Footer', () => {
    const { container } = render(<Footer />);

    expect(container.firstChild).toHaveClass('wrapper')
    expect(container.firstChild.firstChild).toHaveClass('text')
  });

  
});