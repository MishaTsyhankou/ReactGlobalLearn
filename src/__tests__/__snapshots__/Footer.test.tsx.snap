// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should match snapshot 1`] = `
<div>
  <div
    class="wrapper"
  >
    <h2
      class="text"
    >
      <span>
        <b>
          netflix
        </b>
      </span>
      roulette
    </h2>
  </div>
</div>
`;
